index.html

```diff
@@ -1,84 +1,51 @@
 <!DOCTYPE html>
 <html lang="en-us">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>gci15.fossasia.org by fossasia</title>
     <!--Correct the .css path once you have put both files in a folder-->
-    <link rel="stylesheet" type="text/css" href="C:\Users\emanf.ETHANSCOMPUTER\Desktop\FOSSASIA Website\stylesheet.css" media="screen">
+    <link rel="stylesheet" type="text/css" href="stylesheet.css" media="screen">
```

Comment:
Link leads to nowhere. The proper path is **"stylesheets/stylesheet.css"** according to your repository structure